tests/src/test/java/alluxio/client/block/BlockStoreContextIntegrationTest.java

```diff
@@ -91,9 +91,11 @@ public void run() {
    * preventing the release of the worker clients.
    */
   @Test(timeout = 10000)
-  @LocalAlluxioClusterResource.Config(
-      confParams = {PropertyKey.USER_BLOCK_WORKER_CLIENT_THREADS, "10"})
+  @LocalAlluxioClusterResource.Config(startCluster = false)
```

Comment:
as per our offline discussion, it would be great if we can get rid of the `startCluster` flag and always start the cluster